### DISCUSSION TITLE
Publish `adapter-test` package

### DISCUSSION THREAD

**Discussion by hparra** [nextauthjs/next-auth#4130] [Ideas]
### Description 📓

I thought [adapter-test](https://github.com/nextauthjs/next-auth/tree/main/packages/adapter-test) was an actual package but [it doesn't seem so](https://www.npmjs.com/search?q=%40next-auth). It would be nice to have formal testing support for custom adapters.

I'll copy/paste code for now. Thanks for this lib!

### How to reproduce ☕️

```bash
# expected something like the following to work
npm install --save-dev @next-auth/test-adapter
```

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by spaceandrew** [comment]
Second this! Since it's listed in `packages`, definitely thought it was accessible.
